cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -1414,6 +1444,60 @@ define(['jquery', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers', 'common/j
                     );
                 });
 
+                describe('Show correctness setting set as expected.', function() {
+                    var setShowCorrectness = function(show_correctness) {
+                        $('input[name=show-correctness][value=' + show_correctness + ']').click();
+                    }, expectWarningShown = function(shown) {
```

Comment:
@pomegranited aahhh, my eyes :) Could please do it in old-fashined way of

    var setShowCorrectness  = function { ... }
    var expectWarningShown = function { ... }